src/popup/components/TransferSendForm.vue

```diff
@@ -363,12 +365,17 @@ export default defineComponent({
       return nextTick();
     }
 
-    async function queryHandler(query: any) {
-      formModel.value.selectedAsset = availableTokens.value[query.token]
+    async function updateFormModelValues({
+      account,
+      amount,
+      payload,
+      token,
+    }: Dictionary) {
+      formModel.value.selectedAsset = availableTokens.value[token]
         || aeternityToken.value;
-      if (query.account) formModel.value.address = query.account;
-      if (query.amount) formModel.value.amount = query.amount;
-      if (query.payload) formModel.value.payload = query.payload;
+      if (account) formModel.value.address = account;
+      if (amount) formModel.value.amount = amount;
+      if (payload) formModel.value.payload = payload;
```

Comment:
I suggest to wrap the assignments in braces
```suggestion
      if (account) {
        formModel.value.address = account;
      }
      if (amount) {
       formModel.value.amount = amount;
     }
      if (payload) {
        formModel.value.payload = payload;
      }
```